library/ns1_zone

```diff
@@ -212,6 +218,7 @@ def main():
             networks        = dict(required=False, default=None),
             secondary       = dict(required=False, type='dict', default=None),
             primary         = dict(required=False, default=None),
+            dnssec          = dict(required=False, default=None),
```

Comment:
This param's type should be boolean (`type='bool'`) as it does not take arbitrary string input